Dockerfile

```diff
@@ -87,15 +100,24 @@ ARG RUNDIR
 ARG SKIP_TESTS
 ARG SKIP_PRUNE
 
+
 RUN make distclean && make -j ${JOBS} && make $([ "$SKIP_TESTS" != 1 ] && echo runtests) && make clean && make -C ${RUNDIR}
 
 RUN if [ "$SKIP_PRUNE" != 1 ]; then lnls-prune-artifacts ${APP_DIRS} ${PWD} ${RUNDIR}; fi
 
 
 FROM base AS dynamic-link
 
-COPY --from=dynamic-build /opt /opt
+ARG RUNDIR
+ARG SKIP_TESTS
+ARG SKIP_PRUNE
+
+LABEL br.lnls.epics-in-docker.skip-prune=${SKIP_PRUNE}
+LABEL br.lnls.epics-in-docker.app-dirs=${APP_DIRS}
+LABEL br.lnls.epics-in-docker.rundir=${RUNDIR}
```

Comment:
`SKIP_TESTS` wasn't used, and `ADD_DIRS` isn't an `ARG`.

I think the order of args and labels should match, too